great_expectations/datasource/data_connector/configured_asset_azure_data_connector.py

```diff
@@ -0,0 +1,146 @@
+import logging
+import os
+from typing import List, Optional
+
+import great_expectations.exceptions as ge_exceptions
+
+try:
+    from azure.storage.blob import BlobServiceClient
+except ImportError:
+    azure.storage.blob = None  # type: ignore [name-defined]
+
+from great_expectations.core.batch import BatchDefinition
+from great_expectations.core.batch_spec import AzureBatchSpec, PathBatchSpec
+from great_expectations.datasource.data_connector import (
+    ConfiguredAssetFilePathDataConnector,
+)
+from great_expectations.datasource.data_connector.asset import Asset
+from great_expectations.datasource.data_connector.util import list_azure_keys
+from great_expectations.execution_engine import ExecutionEngine
+
+logger = logging.getLogger(__name__)
```

Comment:
@petermoyer had asked for more robust logging; this class has parity with the S3 connectors but could/should we do more?

Reply:
I think that this should be adequate for now (as it is on par with the acceptable pattern throughout the codebase).  In a separate effort, we could do more (e.g., integrate with logging packages and/or services).